<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuCasher</class>
 <widget class="QDialog" name="MenuCasher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="mainLabel">
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Меню Кассира</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Название спектакля</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Количество зрителей:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Количество свободных мест:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="performance">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="countViewer">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="countFreePlace">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonBuyTicket">
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Подтвердить билет</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonHandOverTicket">
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сдать билет</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="accept">
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Принять</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>accept</sender>
   <signal>clicked()</signal>
   <receiver>MenuCasher</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
